requirements/edx/base.txt

```diff
@@ -113,15 +113,15 @@ edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
 edx-django-utils==1.0.3
 edx-drf-extensions==2.1.0
-edx-enterprise==1.3.1
+edx-enterprise==1.3.4
```

Comment:
fyi, i think if you run make upgrade again we are at 1.3.5

Reply:
Good point. I might as well do that here